frontend/src/model/schema/UnionedTable.ts

```diff
@@ -38,4 +40,39 @@ export default class UnionedTable extends BasicTable {
       !prefCol || prefCol.nullable || !altCol || altCol.nullable
     );
   }
+
+  public matchedColumn(column: Column) {
+    const i = this.columns[0].indexOf(column);
+    return this.columns[1][i];
+  }
```

Comment:
sollte das nicht in beide richtungen gehen (also dass man auch columns der rechten seite matchen kann)? und hey, hier würde ein docstring auch niemanden wehtun